frontend/src/components/VariantListPage/VariantListPage.tsx

```diff
@@ -369,6 +396,62 @@ const useVariantListAnnotation = (variantList: VariantList) => {
     [variantList]
   );
 
+  const saveTaggedGroups = useCallback(
+    (taggedGroups) => {
+      patch(`/variant-lists/${variantList.uuid}/shared-annotation/`, {
+        tagged_groups: {
+          A: {
+            displayName: String(taggedGroups.A.displayName),
+            variantList: Array.from(taggedGroups.A.variantList),
+          },
+          B: {
+            displayName: String(taggedGroups.B.displayName),
+            variantList: Array.from(taggedGroups.B.variantList),
+          },
+          C: {
+            displayName: String(taggedGroups.C.displayName),
+            variantList: Array.from(taggedGroups.C.variantList),
+          },
+          D: {
+            displayName: String(taggedGroups.D.displayName),
+            variantList: Array.from(taggedGroups.D.variantList),
+          },
```

Comment:
This could be a good place to use some iteration, that way if we change this in the future its not a hassle. With only 4 examples its not strictly neccesary, but imo its a good habit to get into:

One way of doing that could be using some helpers that JS has to interact with Objects

```
tagged_groups: Object.fromEntries(
	Object.entries(taggedGroups).map([key, group]) => [
		key,
		{
			displayName: String(group.displayName),
			variantList: Array.from(group.variantList),
		}
	])
)
```

Another way would be to use an accumulator:

```
tagged_groups: Object.keys(taggedGroups).reduce((acc, key) => ({
	...acc,
	[key]: {
		displayName: String(taggedGroups[key].displayName),
		variantList: Array.from(taggedGroups[key].variantList),
	}
}), {}
```

This one's slightly more annoying because I think you'd have to type the `{}` at the end to match what you're putting into it